oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/StorageModels.java

```diff
@@ -98,6 +99,7 @@ public Model add(Class<?> aClass, int scopeId, Storage storage, boolean record)
         }
 
         checker.check(storage.getModelName());
+        elasticSearchModelExtension.setRouting(storage.getModelName(), modelColumns);
```

Comment:
```suggestion
        // Set routing rules for ElasticSearch
        elasticSearchModelExtension.setRouting(storage.getModelName(), modelColumns);
        
        checker.check(storage.getModelName());
```

The codes should follow existing code style